src/trajectory.jl

```diff
@@ -146,25 +149,28 @@ transition(
 ### Static trajecotry with fixed leapfrog step numbers.
 ###
 
-struct StaticTrajectory{S<:AbstractTrajectorySampler,I<:AbstractIntegrator} <: AbstractTrajectory{I}
+struct StaticTrajectory{S<:AbstractTrajectorySampler, I<:AbstractIntegrator} <: AbstractTrajectory{I}
     integrator  ::  I
     n_steps     ::  Int
 end
 
-Base.show(io::IO, τ::StaticTrajectory{S,I}) where {I,S<:EndPointTS} =
+function Base.show(io::IO, τ::StaticTrajectory)
     print(io, "StaticTrajectory{EndPointTS}(integrator=$(τ.integrator), λ=$(τ.n_steps)))")
-Base.show(io::IO, τ::StaticTrajectory{S,I}) where {I,S<:MultinomialTS} =
+end
+
+function Base.show(io::IO, τ::StaticTrajectory)
```

Comment:
Again we do need the type info for correct dispatch here.

Reply:
```suggestion
function Base.show(io::IO, τ::StaticTrajectory{<:MultinomialTS})
```